### DISCUSSION TITLE
Is there any way to switch tabs with mouse scroll wheel?

### DISCUSSION THREAD

**Discussion by Esarve** [piroor/treestyletab#3273] [Q&A] [answered]
I saw a feature on sidebery where you can change tabs with scroll wheel, Means you can scroll through the opened tabs. Is it possible in Tree style tabs??

**Comment by Esarve** [comment, marked as answer]
Nevermind, found an extension for it

**Reply by piroor** [reply]
Link for people who came later: Tree Style Tab Mouse Wheel
https://addons.mozilla.org/en-US/firefox/addon/tree-style-tab-mouse-wheel/
